recipes-app/iot2050-eio-manager/iot2050-eio-manager_0.1.bb

```diff
@@ -40,8 +41,12 @@ do_install() {
     install -v -m 755 ${WORKDIR}/iot2050-eio-service.py ${D}/usr/lib/iot2050/eio/
 
     install -v -d ${D}/lib/systemd/system/
+    install -v -m 644 ${WORKDIR}/iot2050-eiofsd.service ${D}/lib/systemd/system/
     install -v -m 644 ${WORKDIR}/iot2050-eiod.service ${D}/lib/systemd/system/
 
     install -v -d ${D}/usr/bin/
+    install -v -m 755 ${WORKDIR}/bin/iot2050-eiofsd ${D}/usr/bin/
```

Comment:
The eiofsd is an internal Siemens project, and due to some reason, it will not be open sourced.

Indeed this will increase the repo size, just like we already have for SEBoot binaries... But I don't think it's easy to find some place eleswhere to hold them...

Reply:
We should clarify licenses - also for the SEBoot binaries as it becomes clear to me only now. Or do we have a statement for the latter already somewhere? Keep in mind that `COPYING.MIT`, the default license, obviously cannot apply.

Reply:
Any idea where to add license clarification of these binaries?

Reply:
Next to the binary or in a top-level file (`COPYING.BINARY` or `COPYING.SIEMENS`?) that lists the paths along with the license text that applies to those files.